docs/x509.rst

```diff
@@ -764,6 +764,40 @@ X.509 Extensions
 
         The serial number of the issuer's issuer.
 
+.. class:: PolicyConstraints
+
+    .. versionadded:: 1.0
+
+    The policy constraints extension can be used in certificates issued
+    to CAs.  The policy constraints extension constrains path validation
+    in two ways.  It can be used to prohibit policy mapping or require
+    that each certificate in a path contain an acceptable policy
+    identifier. For more information about the use of this extension see
+    :rfc:`5280`.
```

Comment:
This should mention that at least one of requires_expliti policy and inhibit policy mapping will be non-None